tests/unit_node/_fs/_fs_handle_test.ts

```diff
@@ -317,3 +317,100 @@ Deno.test({
     await fileHandle.close();
   },
 });
+
+Deno.test(
+  "[node/fs filehandle.createReadStream] Create a read stream",
+  async function () {
+    const fileHandle = await fs.open(testData);
+    const stream = fileHandle.createReadStream();
+    const fileSize = (await fileHandle.stat()).size;
+
+    assertEquals(stream.bytesRead, 0);
+    assertEquals(stream.readable, true);
+
+    let bytesRead = 0;
+
+    stream.on("open", () => assertEquals(stream.bytesRead, 0));
+
+    stream.on("data", (data) => {
+      assertEquals(data instanceof Buffer, true);
+      assertEquals((data as Buffer).byteOffset % 8, 0);
+      bytesRead += data.length;
+      assertEquals(stream.bytesRead, bytesRead);
+    });
+
+    stream.on("end", () => {
+      assertEquals(stream.bytesRead, fileSize);
+      assertEquals(bytesRead, fileSize);
+    });
+
+    // Wait for the 'close' event so that the test won't finish prematurely
+    await new Promise<void>((resolve) => {
+      stream.on("close", resolve);
+    });
+    
+    // await fileHandle.close();
```

Comment:
In both these tests, I had to comment out the `fileHandle.close()` call

If I keep the call, I get

```
error: BadResource: Bad resource ID
    await fileHandle.close();
                     ^
    at FileHandle.close (ext:deno_node/internal/fs/handle.ts:158:33)
    at file:///home/git/deno/tests/unit_node/_fs/_fs_handle_test.ts:350:2
```

I don't know why this error would be thrown here. I think @kt3k is the expert on this part of the code so maybe he has an idea

Reply:
nevermind, I found the issue and left a comment on the fix